style/main.css

```diff
@@ -0,0 +1,230 @@
+*{
+    font-family: 'Montserrat', sans-serif;
+    margin: 0px;
+    padding: 0px;
+}
+
+/* ====== start:content ===== */
+.content {
+    background: #FAFAFA;
+}
+/* ====== end:content ======= */
+
+/* ====== start: Header ====== */
+.header {
+    width: 100%;
+}
+
+.header__item {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 27px 85px;
+}
+.header__content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 10px;
+    padding: 80px 0px;
+}
+.header__sub-content {
+    text-align: center;
+    font-size: 20px;
+    color: #737373;
+}
+.header__action{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 12px;
+    padding: 20px;
+}
+/* ====== end: Header ======== */
+
+/* ===== start:container ===== */
+.container{
+    margin-top: 1rem;
+    padding: 5rem 85px;
+}
+.container--bg-white {
+    background: white;
+}
+/* ===== end:container ======= */
+
+/* ======= start:button ======= */
+.button {
+    border: none;
+    padding: 9px 30px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 28px;
+    cursor: pointer;
+}
+.button--rounded-full {
+    border-radius: 26px;
+}
+.button--primary{
+    background-color: #FFA62B ;
+    border: 1px solid #FFA62B;
+    color: white;
+}
+.button--secondary-outline{
+    border: 1px solid #00A0C1;
+    color: #00A0C1;
+}
+/* ========= end:button ======= */
+
+/* ==== start:site header ==== */
+.site-header{
+    color:#252B42;
+    font-weight: 800;
+}
+.site-header--medium {
+    font-size: 30px;
+}
+.site-header--large {
+    font-size: 50px;
+}
```

Comment:
these modifiers are all for one site-header, how about you create one that has all the modification